alerting/src/main/kotlin/org/opensearch/alerting/workflow/CompositeWorkflowRunner.kt

```diff
@@ -1,30 +1,90 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
 package org.opensearch.alerting.workflow
 
+import org.opensearch.alerting.BucketLevelMonitorRunner
+import org.opensearch.alerting.DocumentLevelMonitorRunner
 import org.opensearch.alerting.MonitorRunnerExecutionContext
+import org.opensearch.alerting.QueryLevelMonitorRunner
 import org.opensearch.alerting.model.MonitorRunResult
+import org.opensearch.alerting.util.isDocLevelMonitor
+import org.opensearch.commons.alerting.model.CompositeInput
 import org.opensearch.commons.alerting.model.Workflow
+import org.opensearch.commons.alerting.util.isBucketLevelMonitor
 import java.time.Instant
+import java.util.UUID
+
+object CompositeWorkflowRunner : WorkflowRunner() {
 
-class CompositeWorkflowRunner : WorkflowRunner() {
     override suspend fun runWorkflow(
         workflow: Workflow,
         monitorCtx: MonitorRunnerExecutionContext,
         periodStart: Instant,
         periodEnd: Instant,
         dryRun: Boolean
-    ): MonitorRunResult<*> {
-        TODO("Not yet implemented")
-    }
+    ): List<MonitorRunResult<*>> {
```

Comment:
This should return workflowRunResult which should contain list of monitorRunResult